rs/nns/governance/src/lib.rs

```diff
@@ -176,17 +176,17 @@ mod subaccount_index;
 /// The value of 10_000 follows the Candid recommendation.
 const DEFAULT_SKIPPING_QUOTA: usize = 10_000;
 
-/// Value: one second after midnight, 2024-11-05 (UTC).
+/// Value: 2024-09-01T00:00:00Z.
 ///
-/// How this value was chosen: This is around the earliest time when
-/// "refreshing" a neuron's voting power might be released, (assuming the usual
-/// NNS release cycle). Significantly different values could also work, but this
-/// seems like a nice "neutral" value.
+/// How this value was chosen: This "pre-ages" "legacy" neurons by a couple of
```

Comment:
How do we define legacy neurons?

Reply:
Here, "legacy" just means those that were created before we introduced the notion of refreshing voting power.